frontend/react-neko-chat/src/styles.css

```
@@ -1523,6 +1523,18 @@ body.neko-electron-runtime .app-shell:not(.chat-surface-mode-compact)[data-focus
   justify-content: center;
 }
 
+.system-chip-source {
+  flex: none;
+  font-size: 11px;
+  letter-spacing: .02em;
+  padding: 1px 6px;
+  border-radius: 999px;
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)